portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,61 @@
+html {
+  font-size: 100%;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
 #greeting-container {
-  margin-top: 20px;
+  margin-top: 1.25em;
 }
+
+.image-thumbnails img {
+    border: 1em solid rgb(132, 179, 249);
+    max-width:100%;
+    max-height:100%;
+}
+
+.image-thumbnails img:hover {
+  border: 1em solid skyblue;
+}
+
+.icons img {
+    border: 0.625em;
+    max-width:10%;
+    max-height:10%;
```

Comment:
nit: Let's add spaces between the ":" and the CSS value so these read like `max-width: 10%` instead of `max-width:10%`

Reply:
Got it.